dataikuapi/dss/modelevaluationstore.py

```diff
@@ -278,80 +286,61 @@ def delete(self):
-
-    def get_file(self, path):
-        """
-        Get a file from the model evaluation
-        
-        Returns:
-            the file's content, as a stream
-        """
-        return self.client._perform_raw(
-                "GET", "/projects/%s/modelevaluationstores/%s/runs/%s/contents/%s" % (self.project_key, self.mes_id, self.run_id, utils.quote(path)))
-
-    def delete_file(self, path):
-        """
-        Delete a file from the model evaluation
-        """
-        return self.client._perform_empty(
-                "DELETE", "/projects/%s/modelevaluationstores/%s/runs/%s/contents/%s" % (self.project_key, self.mes_id, self.run_id, utils.quote(path)))
-
-    def put_file(self, path, f):
-        """
-        Upload the file to the model evaluation
-        
-        Args:
-            f: the file contents, as a stream
-            path: the path of the file
-        """
-        return self.client._perform_json_upload(
-                "POST", "/projects/%s/modelevaluationstores/%s/runs/%s/contents/%s" % (self.project_key, self.mes_id, self.run_id, utils.quote(path)),
-                "", f)
-
     def get_metrics(self):
         """
-        Get the metrics for this model evaluation
+        Get the metrics for this model evaluation. Metrics must be understood here as Metrics in DSS Metrics & Checks
 
         :return: the metrics, as a JSON object
         """
         return self.client._perform_json(
             "GET", "/projects/%s/modelevaluationstores/%s/runs/%s/metrics" % (self.project_key, self.mes_id, self.run_id))
 
-class DSSModelEvaluationSettings:
-    """
-    A handle on the settings of a model evaluation
 
-    Do not create this class directly, instead use :meth:`dataikuapi.dss.DSSModelEvaluation.get_settings`
+class DSSModelEvaluationFullInfo:
     """
+    A handle on the full information on a model evaluation.
 
-    def __init__(self, model_evaluation, settings):
+    Includes information such as the full id of the evaluated model, the evaluation params,
+    the performance and drift metrics, if any, etc.
+
+    Do not create this class directly, instead use :meth:`dataikuapi.dss.DSSModelEvaluation.get_full_info`
+    """
+    def __init__(self, model_evaluation, full_info):
         self.model_evaluation = model_evaluation
-        self.settings = settings
-        # unpack some fields
-        self.client = model_evaluation.client
-        self.run_id = model_evaluation.run_id
-        self.project_key = model_evaluation.project_key
-        self.mes_id = model_evaluation.mes_id
+        self.full_info = full_info
 
     def get_raw(self):
-        return self.settings
+        return self.full_info
 
-    def save(self):
-        return self.client._perform_json(
-                "PUT", "/projects/%s/modelevaluationstores/%s/runs/%s/settings" % (self.project_key, self.mes_id, self.run_id),
-                        body=self.settings)
+    def get_metrics(self):
```

Comment:
I find this a bit confusing:

```
me.get_full_info().get_metrics()
```
and
```
me.get_metrics()
```

... don't return the same thing and they use a different format

However I don't really see what you can do to clarify that, given how it works under the hood. Maybe one of them you be hidden/"less documented" ? (the 1st is definitely what people will want in practice)